napari/_qt/layer_controls/qt_image_controls.py

```diff
@@ -167,12 +166,13 @@ def __init__(self, layer) -> None:
         self.isoThresholdSlider = sld
         self.isoThresholdLabel = QLabel(trans._('iso threshold:'))
 
-        sld = QSlider(Qt.Orientation.Horizontal, parent=self)
+        sld = QDoubleSlider(Qt.Orientation.Horizontal, parent=self)
```

Comment:
```suggestion
        sld = QLabeledDoubleSlider(Qt.Orientation.Horizontal, parent=self)
```
This should fix the failing `min_req` test, because the correct slider will be explicitly used.